t/after.t

```diff
@@ -0,0 +1,20 @@
+use strict;
+use warnings;
+
+use Test::More;
+use Path::Tiny;
+use Rex::Commands;
+
+$::QUIET = 1;
+
+my $tmpfile = Path::Tiny->tempfile;
```

Comment:
I mean it's important because most of these non-core modules are often not available on different OSes, and it makes it hard to impossible for package maintainers to include Rex in different distros. And/or in those cases, _we_ have to package and maintain those dependencies in our own package repositories, which means lots of sustained overhead. And/or those modules are not compatible with perl back to 5.8.9 (which we still have to support, *sigh*).

IMHO all of the above can be avoided by spending some extra minutes on coding instead :)